pkg/mutate/manage_pod_mutation.go

```diff
@@ -0,0 +1,132 @@
+package mutate
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+
+	offv1alpha1 "github.com/liqotech/liqo/apis/offloading/v1alpha1"
+	liqoconst "github.com/liqotech/liqo/pkg/consts"
+)
+
+// getVirtualNodeToleration returns a new Toleration for the Liqo's virtual-node.
+func getVirtualNodeToleration() corev1.Toleration {
+	return corev1.Toleration{
+		Key:      liqoconst.VirtualNodeTolerationKey,
+		Operator: corev1.TolerationOpExists,
+		Effect:   corev1.TaintEffectNoExecute,
+	}
+}
+
+// createTolerationFromNamespaceOffloading creates a new virtualNodeToleration in case of LocalAndRemotePodOffloadingStrategyType
+// or RemotePodOffloadingStrategyType. In case of LocalPodOffloadingStrategyType returns an empty Toleration.
+func createTolerationFromNamespaceOffloading(strategy offv1alpha1.PodOffloadingStrategyType) corev1.Toleration {
+	var toleration corev1.Toleration
+	switch {
+	case strategy == offv1alpha1.LocalAndRemotePodOffloadingStrategyType, strategy == offv1alpha1.RemotePodOffloadingStrategyType:
+		// The virtual-node toleration must be added.
+		toleration = getVirtualNodeToleration()
+	default:
```

Comment:
In my opinion, we should add a case for `LocalPodOffloadingStrategyType` and print and error in the default case.

This should prevent future errors, for example, if we will add other strategies and we will forget to handle them in this switch.

Reply:
Great idea, thank you